pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1005,6 +1005,11 @@
             doc = "List of interceptors for entry metadata.")
     private Set<String> brokerEntryMetadataInterceptors = new HashSet<>();
 
+    @FieldContext(
+            category = CATEGORY_SERVER,
+            doc = "Enable or disable exposing broker entry metadata to client.")
+    private boolean enableExposingBrokerEntryMetadataToClient = false;
```

Comment:
`exposingBrokerEntryMetadataToClientEnabled` is better

Reply:
Thanks. Fixed.